data/analytics/events.yml

```diff
@@ -552,6 +555,9 @@
       implemented: true
```

Comment:
This event is meant for links to gov.uk files which aren't in an attachment component, for example on this page: https://www.gov.uk/government/news/the-2023-government-chemist-conference-presentations the "Welcome address- Julian Braybrook" link.

For attachment components, there are two different `event_name`s that an attachment link can have. If it's a HTML attachment, it has `event_name: navigation`, and if it's a file, it has `event_name: file_download`.
These seem to be covered already by http://127.0.0.1:4567/analytics/event_html_attachments.html and http://127.0.0.1:4567/analytics/event_links.html#heading-3 (File attachment links)

Would you be able to make this event match the "Welcome address- Julian Braybrook" link? Thanks

Reply:
Thanks @AshGDS, have updated the `Update links GA4 event data` commit with these changes, let me know if it's alright.